tests/lib/eslint.js

```diff
@@ -2488,6 +2488,23 @@ describe("eslint", function() {
             assert.equal(messages[2].column, 18);
         });
 
+        it("should accept a pregenerated ast", function() {
+
+            var code = "foo()\n    alert('test')";
+            var config = { rules: { "no-mixed-spaces-and-tabs": 1, "eol-last": 1, "semi": [1, "always"] } };
+
+            var ast = eslint.parse(code, config);
+
+            var messages = eslint.verify({text: code, ast: ast}, config, filename);
```

Comment:
By passing in an object literal as the first parameter, I think you are not really passing a true result from parsing.  For instance, the tokens are missing.  I think a better test would be to use the espree parser and pass the result into verify.